libs/feature/search/src/lib/favorites/favorite-star/favorite-star.component.html

```diff
@@ -5,7 +5,7 @@
   <gn-ui-star-toggle
     [toggled]="isFavorite$ | async"
     (newValue)="toggleFavorite($event)"
-    [disabled]="loading || (isAnonymous$ | async)"
```

Comment:
the problem is that it gives a weird user experience; thanks for pointing out this new problem

Reply:
I made sure that the star was clickable again after the request ended; this gives a slightly better UX; the whole thing is not working correctly anyway because of auth issues with GN...